src/MCPClient/debian/control

```diff
@@ -45,5 +45,6 @@ Depends:
 	tree,
 	ufraw,
 	unrar-free,
+	python-unidecode,
```

Comment:
Without it, the sanitizeObjectNames microservice fails with

`
Traceback (most recent call last):
  File "/usr/lib/archivematica/MCPClient/clientScripts/sanitizeObjectNames.py", line 38, in <module>
    import sanitizeNames
  File "/usr/lib/archivematica/MCPClient/clientScripts/sanitizeNames.py", line 30, in <module>
    from unidecode import unidecode
ImportError: No module named unidecode`

I added it as a package dependency, to avoid messing with requirements.txt, but I'll do the opposite to make packaging simpler.